### PR TITLE
Added the apology msg and links

### PR DESCRIPTION
**Pull request by tharun143**
Signed-off-by: tharun143 <tharunravuri2000@gmail.com>

### CODE REVIEW

**Comment by tharun143** [comment]
@klumb Is this fine?

**Comment by klumb** [comment]
This looks good. I am merging it.

File: CHAOSScon/2020NA/event-details.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+
 ### CHAOSScon NA 2020 is canceled
 
+
+Thank you for your interest in CHAOSScon North America 2020. After having discussions with the communnity, we regret to inform that the CHAOSScon North America 2020 was cancelled due to the COVID-19 pandemic. We truly appreciate your interest in CHAOSScon. We are encouraging people to participate through alternate ways.
+
+- [Youtube channel](https://www.youtube.com/c/CHAOSStube)
+- [Twitter](https://www.youtube.com/c/CHAOSStube)
```
Comment (GeorgLink) [inline CHAOSScon/2020NA/event-details.md:8]: The twitter link needs to be fixed.

**Comment by GeorgLink** [comment]
I see, this was already merged. I'll create a PR for this

**Pull request merged**